src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
```

Comment:
Please stick to BEM metodology. Here it's better to change class names. Also, remember about proper HTML formatting.

```suggestion
      <nav class="navigation">
        <ul class="navigation__list">
          <li class="navigation__item">
            <a href="#" class="navigation__link is-active">
              Apple
            </a>
          </li>
```

Reply:
Also, it's better to use is-active as BEM modifier 
```suggestion
            <a href="#" class="navigation__link--is-active">Apple</a>
```